test/integration/targets/openssl_certificate/tasks/ownca.yml

```diff
@@ -116,4 +116,15 @@
     issuer:
       commonName: Example CA
 
+- name: Create ownca certificate with notBefore and notAfter
+  openssl_certificate:
+    provider: ownca
+    ownca_not_before: 20181023133742Z
+    ownca_not_after: 20191023133742Z
```

Comment:
Have you (or anyone else) tested what happens when `ownca_not_after` lies in the past? Does OpenSSL complain, or just signs the certificate? I'm wondering because if it complains / dies / ..., this test will start failing in a year. :)

Reply:
@felixfontein good point! I just went ahead and tested it, the certificate is created without problems when choosing a validity period in the past. It even succeeds when notAfter lies before notBefore.

Reply:
Awesome! That's what I was hoping for! :)